spec/_attachments/ic.did

```diff
@@ -113,6 +113,14 @@ type send_transaction_request = record {
 
 type millisatoshi_per_byte = nat64;
 
+type snapshot_id = nat64;
```

Comment:
I remember the recommendation that we should be using `nat` for all numbers unless there's a strong reason not to. @chenyan-dfinity can you confirm if that's correct?

Reply:
Another minor point is that `nat` is more space efficient on the wire. `nat64` always takes 8 bytes, while `nat` can have fewer bytes if the actual number is small.